From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Thu, 31 Aug 2017 07:11:39 -0400
Subject: Skip failing tests

That way we can still benefit by running all the other tests.

The actor-offscreen-redirect test still fails on mips but passes
on other arches as of September 2018.
https://buildd.debian.org/status/logs.php?pkg=mutter&ver=3.30.0-4

The actor-shader-effect test failed on mipsel with 3.30.0-5
after passing with 3.30.0-4
https://buildd.debian.org/status/logs.php?pkg=mutter&ver=3.30.0-5

Note that actor-shader-effect does fail in clutter-1.0.

There was one report of both those tests being flaky on i386
but I wasn't able to reproduce.
See https://launchpad.net/bugs/1795556

Bug: https://bugzilla.gnome.org/785319
Bug: https://bugzilla.gnome.org/782215
Applied-upstream: no
---
 clutter/tests/conform/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/clutter/tests/conform/Makefile.am b/clutter/tests/conform/Makefile.am
index 9cef08b..0dec96e 100644
--- a/clutter/tests/conform/Makefile.am
+++ b/clutter/tests/conform/Makefile.am
@@ -26,10 +26,8 @@ actor_tests = \
 	actor-layout \
 	actor-meta \
 	actor-offscreen-limit-max-size \
-	actor-offscreen-redirect \
 	actor-paint-opacity \
 	actor-pick \
-	actor-shader-effect \
 	actor-size \
 	$(NULL)
 
